sample/mpp-library/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
     commonMainApi(projects.units)
 
     "androidMainImplementation"(libs.recyclerView)
+    "androidMainImplementation"(projects.unitsDatabinding)
```

Comment:
no, we should extract databinding logic from mpp-library at all. also we should remove kapt plugin from mpp and remove javaxAnnotation from mpp

Reply:
removed kapt, javaxAnnotation, and databinding enabled from mpp